### ISSUE TITLE
How to actually install on Linux?

### ISSUE THREAD

**Issue by Fabian42** [MrS0m30n3/youtube-dl-gui#349]
How do I install this on Linux?
1. It requires Python 2.7.3+, fine. When I follow the link, I get only a source, which gives errors when trying to install, but I don't really know which ones, since I get a gazillion lines of output. But I have version 2.7.13-2 of the `python` package installed, it's shown in Synaptic. So I guess I'm ok?
2. It requires wxPython 3. When I follow the link, I get a lengthy text how Linux isn't really supported and there are many problems with it. Not good. But let's continue:
3. It requires TwoDict. When I follow that link, I get an archive and installation instructions, everything works fine.
4. It requires GNU gettext, which requires other stuff, all of that gives some error messages, but in the end it seems to be done, I guess? I don't know, thousands of lines of console output flew past the screen, I have no idea what they said.
5. It requires FFmpeg. When I follow the link, I get something where not even `./configure` works, but whatever, I should have at least one ffmpeg version leftover from previous troubles with another program.

Then I want to install it. I download "Source (.zip)", unpack it and do `python setup.py install` in it, I get `No module named wx`. So wxPython is apparently missing. But I also can't install it, because whatever.

So I try the Windows versions with Wine. With both I get an error and the log says:
```Traceback (most recent call last):
  File "youtube-dl-gui", line 35, in <module>
  File "youtube_dl_gui\__init__.pyc", line 56, in <module>
  File "youtube_dl_gui\utils.pyc", line 189, in get_config_path
  File "ntpath.pyc", line 65, in join
  File "ntpath.pyc", line 115, in splitdrive
TypeError: object of type 'NoneType' has no len()```

So, in short, everything is complicated and annoying. Am I missing something? Is there an installation script somewhere or a clear list of commands to execute? I have no problem installing a list of dependencies, but if the dependencies all have to be built from source, have their own problems and dependencies which have to be built from source and so on, it's starting to get ridiculous.

So how do I actually install it? What are the proper instructions?

**Comment by Axel-Erfurt** [comment]
> It requires wxPython 3.

??

I think that's not true

**Comment by Fabian42** [comment]
https://github.com/MrS0m30n3/youtube-dl-gui#requirements
